app/views/content_items/consultation.html.erb

```diff
@@ -3,14 +3,17 @@
     <%= render 'govuk_component/title', @content_item.title_and_context %>
   </div>
   <%= render 'shared/translations' %>
+</div>
+
+<%= render 'shared/publisher_metadata_with_logo', content_item: @content_item %>
+
+<div class="grid-row">
   <div class="column-two-thirds">
     <%= render 'components/notice', @content_item.withdrawal_notice_component  %>
```

Comment:
These notices will be 2/3rds width which doesn't seem consistent? Is this a special case?

Reply:
Ah that's one I missed - will fix now 👍